code/modules/religion/burdened/psyker.dm

```diff
@@ -0,0 +1,305 @@
+		"Rod of God" = "Splitting the red sea again.",
+		"Holy Grail" = "You found it!",
+		"Burning Bush" = "Useful for any burning ambush.",
+		"Judgement" = "First of all, damn. Alpha much? Dude, so cool, and so are you! Strong, too!",
+		"Paradiso" = "A divine end to the comedy of life.",
+		"DVNO" = "Don't need to ask my name to figure out how cool I am.",
+		"Venus Supermax" = "Did you know nearly everyone working and living on Venus is involved in sulfur extraction? Quite fitting for this weapon of gunpowder.",
+		"Nirvana" = "The giver of quietude, freedom, and highest happiness.",
+		"Cerebrum Dispersio" = "Latin for \"brain splitting\". How fitting.",
+		"Ultimort" = "Your hope dies last.",
+		"Lifelight" = "No escape, no greater fate to be made.",
+		"Bendbreaker" = "FRAGILE: Please do not bend or break.",
+		"Pop Pop" = "The name referring to an onomatopeia (phonetic imitation) of a gun firing.",
+		"Justice" = "Justice is Splendor.",
+		"Splendor" = "Splendor is Justice.",
+		"Revelation" = "Awaken your faith.",
+		"New Safety M62" = "This model of firearm is popular hundreds of years later due to masculine associations created by the film industry.",
+		"Unmaker" = "What the !@#%* is this!",
+		"INKVD" = "Savior of the soul and fighter against dirty thoughts.",
+		"Life Leech" = "An artifact said to draw its power from the life energy of others.",
+		"Nullray" = "Starless metal on the barrel imbibes light and routes it to the null place. The grip acrylic is patterned after ley lines.",
+		"Mortis" = "Put your faith into this weapon working.",
+		"Ramiel" = "Literally meaning \"God has thundered\". You could even interpret the gunshot as a thunder.",
+		"Daredevil" = "Hey now, you won't be reckless with this, will you?",
+		"Lacytanga" = "Rules are written by the strong.",
+		"A10" = "The fist of God. Keep away from the terrible.",
+	)
+
+/obj/item/gun/ballistic/revolver/chaplain/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE_HOLY|MAGIC_RESISTANCE_MIND)
+	name = pick(possible_names)
+	desc = possible_names[name]
+
+/obj/item/gun/ballistic/revolver/chaplain/attack_self(mob/living/user)
+	pray_refill(user)
+
+/obj/item/gun/ballistic/revolver/chaplain/suicide_act(mob/living/user)
+	. = ..()
+	name = "Habemus Papam"
+	desc = "I announce to you a great joy."
+
+/obj/item/gun/ballistic/revolver/chaplain/proc/pray_refill(mob/living/carbon/human/user)
+	if(DOING_INTERACTION_WITH_TARGET(user, src) || !istype(user))
+		return
+	var/datum/brain_trauma/special/burdened/burden = user.has_trauma_type(/datum/brain_trauma/special/burdened)
+	if(!burden || burden.burden_level < 9)
+		to_chat(user, span_warning("You aren't burdened enough."))
+		return
+	user.manual_emote("presses [user.p_their()] palms together...")
+	if(!do_after(user, 5 SECONDS, src))
+		balloon_alert(user, "interrupted!")
+		return
+	user.say("#Oh great [GLOB.deity], give me the ammunition I need!", forced = "ammo prayer")
```

Comment:
if they're at burden_level 9 or above, they should have the unintelligible mutation, no?

Reply:
yes its pretty funny